### DISCUSSION TITLE
I Feel Like We're Missing Out.

### DISCUSSION THREAD

**Discussion by Overreationary** [3kh0/ext-remover#551] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I have figured out how to use the Point Blank expoilt to run explots on privliaged pages, but I feel like instead of turning an extention on and off and on and off, we should have a code that just turns it off. Am I being dumb, or is there just a simple code for this? I haven't been able to figure it out.

**Comment by Aka-but-nice** [comment]
> I have figured out how to use the Point Blank expoilt to run explots on privliaged pages, but I feel like instead of turning an extention on and off and on and off, we should have a code that just turns it off. Am I being dumb, or is there just a simple code for this? I haven't been able to figure it out.

Like, just to straight up turn it off?

**Reply by Overreationary** [reply]
Yes

**Comment by Overreationary** [comment]
Yup

**Reply by cooltoo37** [reply]
then what is the code

**Comment by siouxxsies** [comment, marked as answer]
LTBEEF and INGOT were those who would turn off extensions but they don't work on chromebooks above 108 which SUCKS. 

hopefully new exploits come out </3
